user_config.c

```diff
@@ -74,7 +74,7 @@ void parse_usb_string(const char *input) {
                             break;
                         }
 
-                    } else if (!sid_sent && data_index < 5) {
+                    } else if (!sid_sent && data_index < 10) {
                         data_to_send[data_index] = char_to_check;
                         data_index++;
                         break;
```

Comment:
does this condition and the one below it do the same thing? Also I'd use `sizeof(data_to_send) - 1` (if that's what 10 is here).